tox.ini

```diff
@@ -13,15 +13,15 @@
 #  and also to help confirm pull requests to this project.
 
 [tox]
+minversion = 3.7.0
```

Comment:
Please add a comment here indicating that this is the `tox` minimum version and why it's necessary.